vignettes/ab-testing.Rmd

```diff
@@ -0,0 +1,739 @@
+---
+title: "Probabilistic A/B Testing with rstanarm"
+author: "Imad Ali"
+date: "`r Sys.Date()`"
+output:
+  html_vignette:
+    toc: yes
+params:
+  EVAL: !r identical(Sys.getenv("NOT_CRAN"), "true")
```

Comment:
We can't use the `params` mechanism any more because some OS does not have a sufficiently recent Pandoc or something. The way we are doing the other vignettes now is to put `knitr::opts_chunk$set(eval = identical(Sys.getenv("NOT_CRAN"), "true")))`

Reply:
I got around this in other packages just by checking `if (exists("params"))` but not using it is also fine.

Reply:
Ok, I'll just remove it then.